externals/kyuubi-hive-sql-engine/pom.xml

```diff
@@ -159,63 +165,63 @@
-                        <include>com.fasterxml.jackson.core:jackson-annotations</include>
-                        <include>com.fasterxml.jackson.core:jackson-core</include>
-                        <include>com.fasterxml.jackson.core:jackson-databind</include>
-                        <include>com.fasterxml.jackson.module:jackson-module-scala_2.12</include>
-                        <include>org.apache.kyuubi:kyuubi-common_${scala.binary.version}</include>
-                        <include>org.apache.kyuubi:kyuubi-events_${scala.binary.version}</include>
-                        <include>org.apache.kyuubi:kyuubi-ha_${scala.binary.version}</include>
-                        <include>org.apache.curator:curator-client</include>
-                        <include>org.apache.curator:curator-framework</include>
-                        <include>org.apache.curator:curator-recipes</include>
-                    </includes>
-                </artifactSet>
-                <filters>
-                    <filter>
-                        <artifact>*:*</artifact>
-                        <excludes>
-                            <exclude>META-INF/*.SF</exclude>
-                            <exclude>META-INF/*.DSA</exclude>
-                            <exclude>META-INF/*.RSA</exclude>
-                            <exclude>META-INF/DEPENDENCIES</exclude>
-                            <exclude>META-INF/LICENSE.txt</exclude>
-                            <exclude>META-INF/NOTICE.txt</exclude>
-                            <exclude>META-INF/maven/**</exclude>
-                            <exclude>LICENSE.txt</exclude>
-                            <exclude>NOTICE.txt</exclude>
-                            <exclude>mozilla/**</exclude>
-                            <exclude>**/module-info.class</exclude>
-                        </excludes>
-                    </filter>
-                </filters>
-                <relocations>
-                    <relocation>
-                        <pattern>org.apache.curator</pattern>
-                        <shadedPattern>${kyuubi.shade.packageName}.org.apache.curator</shadedPattern>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <configuration>
+                    <shadedArtifactAttached>false</shadedArtifactAttached>
+                    <artifactSet>
                         <includes>
-                            <include>org.apache.curator.**</include>
+                            <include>org.scala-lang:scala-library</include>
+                            <include>com.fasterxml.jackson.core:jackson-annotations</include>
+                            <include>com.fasterxml.jackson.core:jackson-core</include>
+                            <include>com.fasterxml.jackson.core:jackson-databind</include>
+                            <include>com.fasterxml.jackson.module:jackson-module-scala_2.12</include>
```

Comment:
Not related, but seems we include jackson w/o relocating, it may cause potential risk for class confliction.

I don't see content diff, it's just a format change? If yes, please revert this part to minimize the diff of this PR, we can handle it in seperated PR.

Reply:
A separate PR is better

Reply:
rollback this change